### PR TITLE
Fixed an issue with sample table load auto-updates.

### PR DESCRIPTION
**Pull request by hepcat72**
## Summary Change Description

Replaced the buffered update U erroneously removed 8 commits ago.

## Affected Issue Numbers

- Resolves no issue

## Code Review Notes

self explanatory.  I should probably add a test, but I gotta run.

## Checklist

- [x] All issue requirements satisfied (or no linked issues)
- [x] [Linting passes](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#linting).
- [x] [Migrations created & committed *(or no model changes)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#migration-process)
- Tests
  - [x] [Tests implemented *(or no code changes)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#test-implementation)
  - [x] [All *multi_working* tag tests pass locally](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#quality-control)
  - [x] All example load tests pass (remotely) *(or their failures predated and are unrelated to this branch)*
  - [x] `@tag("multi_working")` added to newly passing test functions/classes *(or no newly passing tests)*
  - [x] `@tag("multi_broken")`, `@tag("multi_unknown")`, or `@tag("multi_mixed")` removed from newly passing test functions/classes *(or no newly passing tests)*

### CODE REVIEW

**Review by lparsons — CHANGES_REQUESTED** [review]
I'll trust that this is necessary and correct. Linting issues should be fixed before merging. Doesn't require re-review assuming those are the only changes.

**Comment by hepcat72** [comment]
> I'll trust that this is necessary and correct. Linking issues should be fixed before merging. Doesn't require re-review assuming those are the only changes.

Well this is interesting... The failed tests pass for me locally. I wonder if it could be due to my python version: `3.9.7`. I fixed the linting errors and will push again to see if this was a fluke...

**Review by fkang-pu — APPROVED** [review]
I created a local branch based on this PR, and ran some data loading and tests.  Fired up runserver and the web pages worked. The tests with the following tags also worked for me. 
The "multi_mixed" and "multi_broken" tags could be removed if the tests passed on your end.

```
python manage.py test --tag=multi_broken
python manage.py test --tag=multi_mixed
python manage.py test --tag=qs2df
```

**Comment by hepcat72** [comment]
> I created a local branch based on this PR, and ran some data loading and tests. Fired up runserver and the web pages worked. The tests with the following tags also worked for me. The "multi_mixed" and "multi_broken" tags could be removed if the tests passed on your end.
> 
> ```
> python manage.py test --tag=multi_broken
> python manage.py test --tag=multi_mixed
> python manage.py test --tag=qs2df
> ```

Thanks Fan, that's helpful. What version of python do you run locally?  Do you get any errors if you run `python manage.py test DataRepo.tests.test_views.ViewTests`?

**Comment by fkang-pu** [comment]
> Thanks Fan, that's helpful. What version of python do you run locally? Do you get any errors if you run `python manage.py test DataRepo.tests.test_views.ViewTests`?

Python 3.9.1
Just ran the test you mentioned above, no errors.

File: .github/workflows/tracebase-tests.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ jobs:
       - name: Set up Python 3.8
         uses: actions/setup-python@v1
         with:
-          python-version: 3.8
+          python-version: 3.9
```
Comment (lparsons) [inline .github/workflows/tracebase-tests.yml:55]: We are running Python 3.8 on the TraceBase servers, so we need to ensure things run properly on that version.

Comment (lparsons) [reply .github/workflows/tracebase-tests.yml:55]: FWIW, the errors don't look related to python version.

One example error is:
```
2022-09-15T18:00:38.9991880Z DataRepo.utils.exceptions.MissingSamplesError: 2 samples are missing: blank_1_404020, blank_15N_AAmix_10Xstd
```

In total:
```
2022-09-15T18:00:40.3700777Z Ran 212 tests in 847.326s
2022-09-15T18:00:40.3700898Z 
2022-09-15T18:00:40.3700996Z FAILED (failures=1, errors=23)
```

**Comment by hepcat72** [comment]
All the (`multi_working`) tests now pass.  Waiting for the rerun of linting.  All I did was insert a space.  I added new `multi_broken` tagged tests for a problem defined in issue #513.  I would like @fkang-pu's help to get those tests to pass, but I think that this is now good to merge and push to dev.  I will merge shortly.

**Pull request merged**